### ISSUE TITLE
thank_you section contrast ratio is below WCAG 2.0 guidelines.

### ISSUE THREAD

**Issue by vishalx360** [fossasia/gci18.fossasia.org#213]
**Actual Behaviour**
- thank_you section has contrast ratio of **`1.71`** which is below **WCAG 2.0 _[guidelines](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html#visual-audio-contrast-contrast-resources-head)_**. 
**WCAG 2.0** guidelines dictate a minimum **contrast ratio** of **`4.5:1`** for the visual presentation of both text and images embedded as text.

- thank_you section has **inconsistent** padding as compared to above all elements.
**at**`
@media (max-width: 767px)
section {
    padding: 80px 0; 
}
`
and
**at**`
section, footer {
    padding: 96px 0;
    position: relative;
    overflow: hidden;
}
`

**Expected Behaviour**

- it should have higher contrast ratio for **better visibility.**
- it should have **consistent** padding and margin for **better semantics**

**Steps to reproduce it**

- make background gradient of **`div#thankyou`** darker according to ratio of foreground
- remove `padding: 96px 0;` with `margin-top: 20px;` at **sections** part of theme.css

**LogCat for the issue**

**Screenshots of the issue**
- contrast ratio of `1.71`
![selection_066](https://user-images.githubusercontent.com/25267239/47963996-c369c480-e059-11e8-9336-d158de0c3643.png)

- **inconsistent padding** _top and bottom_ 

![gci18 fossasia org_ iphone 6_7_8 plus 2](https://user-images.githubusercontent.com/25267239/47964329-79cfa880-e05e-11e8-9bb0-df1398156f1e.png)

**Would you like to work on the issue?**

  **yah Sure**
